plugins/zoom/icon_zoom/connection/connection.py

```diff
@@ -12,17 +12,20 @@ def __init__(self):
         self.zoom_api = None
 
     def connect(self, params):
-        self.logger.info("Connect: Connecting...")
-        api_key = params.get(Input.API_KEY).get("secretKey")
-        secret = params.get(Input.SECRET).get("secretKey")
+        SECRETKEY = "secretKey"
 
-        self.zoom_api = ZoomAPI(api_key, secret, self.logger)
+        account_id = params[Input.ACCOUNT_ID][SECRETKEY]
+        client_id = params[Input.CLIENT_ID][SECRETKEY]
+        client_secret = params[Input.CLIENT_SECRET][SECRETKEY]
```

Comment:
To avoid `Key Errors`
```suggestion
        account_id = params.get(Input.ACCOUNT_ID, {}).get(SECRETKEY, "")
        client_id = params.get(Input.CLIENT_ID, {}).get(SECRETKEY, "")
        client_secret = params.get(Input.CLIENT_SECRET, {}).get(SECRETKEY, "")
```

Reply:
I had originally went with `.get()`, but I decided against it since it should fail earlier due to those inputs all being required. I'm fine with using `.get()` though!